client/state/automated-transfer/selectors/is-automated-transfer-active.js

```diff
@@ -1,14 +1,15 @@
 /**
  * External dependencies
  */
-
 import { flowRight as compose } from 'lodash';
 
 /**
  * Internal Dependencies
  */
 import { transferStates } from 'state/automated-transfer/constants';
-import { getAutomatedTransferStatus } from 'state/automated-transfer/selectors';
+import { getAutomatedTransferStatus } from './get-automated-transfer-status';
```

Comment:
Any reason why we don't import from `state/automated-transfer/selectors` here? 

Also, as you know I'd suggest always using absolute import paths 😉 

Both of these can be applied on multiple occasions in this PR.

Reply:
I don't want to import from `state/automated-transfer/selectors` as I'd rather avoid cyclical dependencies where possible. They're usually not a problem in the end, but it makes the dependency graph messy.

I'll switch to absolute paths, though! This PR was prepared before our previous discussion 😄

Reply:
Done (and rebased on master).